WordPress/jetpack_metadata/release_notes.txt

```diff
@@ -1,10 +1,7 @@
-* [*] Fixed email app detection in signup for Android 11, and removed the useless button when none are available. [https://github.com/wordpress-mobile/WordPress-Android/pull/15911]
 * [*] Comment editor: Notification comments support in the new editor [https://github.com/wordpress-mobile/WordPress-Android/pull/15884]
 * [*] Site creation: Fixed bug where sites created within the app were not given the correct time zone, leading to post scheduling issues. [https://github.com/wordpress-mobile/WordPress-Android/pull/15904]
 * [*] Login Epilogue: Fixed bug where if no sites available, then new login epilogue screen without clickable sites and with "create new site" is shown instead of no sites empty "My Site" view. [https://github.com/wordpress-mobile/WordPress-Android/pull/15944]
 * [*] Fixes flickering when changing the preview mode in Page or Site design picker [https://github.com/wordpress-mobile/WordPress-Android/pull/15943]
 * [**] Jetpack App: Signup using WordPress.com, create your favorite site and enjoy the Reader inside Jetpack app. [https://github.com/wordpress-mobile/WordPress-Android/pull/15946]
-* [*] Reader: Admins and Editors can edit comments from Reader [https://github.com/wordpress-mobile/WordPress-Android/pull/15957]
```

Comment:
@ashiagr Likewise, should I restore this entry from the JP release notes?

Reply:
Since `Reader` was never enabled for JP until now, we can remove this entry for JP users as they need not know if anything was fixed for Reader.

Reply:
Addressed in d07d573c07. Thanks for the feedback!